close_test.go

```diff
@@ -433,14 +432,16 @@ func TestCloseSendError(t *testing.T) {
 	clientCh := testutils.NewClient(t, nil)
 
 	// Create a connection that will be shared.
-	require.NoError(t, testutils.Ping(clientCh, serverCh), "Ping from client to server failed")
+	// Use PingLong here so that we have a context with a bigger timeout
```

Comment:
let's remove `PingLong` and `EchoLong` away from testutils and just have one context created with a long timeout here and call them directly. I'd like to avoid `testutils` having methods that are only used by a single call site.